File: src/test/java/solutions/n733_flood_fill/SolutionTest.java
package solutions.n733_flood_fill;

import org.junit.Assert;
import org.junit.Test;

/**
 * @author : Sobornov Vladimir
 * @since : 02.12.2022
 **/

public class SolutionTest {

    int[][] test2DArray = { { 1, 1, 1 }, { 1, 1, 0 }, { 1, 0, 1 } };
    int[][] test2DArrayAssert = { { 2, 2, 2 }, { 2, 2, 0 }, { 2, 0, 1 } };

    @Test
    public void floodFillTest() {
        Assert.assertEquals(test2DArrayAssert , Solution.floodFill(test2DArray, 1, 1, 2));
    }
}

File: src/test/kotlin/solutions/n4_median_of_two_sorted_arrays/SolutionTest.kt
package solutions.n4_median_of_two_sorted_arrays

import org.junit.Assert
import org.junit.Test

class SolutionTest {
    private val solution = Solution();

    @Test
    fun findMedianSortedArraysTest1() {
        Assert.assertEquals(2.00000, solution.findMedianSortedArrays(intArrayOf(1,3), intArrayOf(2)), 10E-5)
    }

    @Test
    fun findMedianSortedArraysTest2() {
        Assert.assertEquals(2.50000, solution.findMedianSortedArrays(intArrayOf(3,4), intArrayOf(1,2)), 10E-5)
    }
}